salt/_modules/metalk8s.py

```diff
@@ -181,61 +181,57 @@ def product_info_from_iso(path):
             )
         )
 
-    return _get_product_info(result['stdout'])
+    return _get_archive_info(result['stdout'])
 
 
-def get_products(products=None):
-    """Get a matching between version and path from products or
-    `metalk8s.products` from pillar if products is None
+def get_archives(archives=None):
+    """Get a matching between version and path from archives or
+    `metalk8s.archives` from pillar if archives is None
 
     Arguments:
-        products (list): list of path to directory or iso
+        archives (list): list of path to directory or iso
     """
-    if not products:
-        products = __pillar__.get('metalk8s', {}).get('products', [])
-
-    if isinstance(products, unicode):
-        products = str(products)
-    if isinstance(products, str):
-        products = [products]
-    if not isinstance(products, list):
+    if not archives:
+        archives = __pillar__.get('metalk8s', {}).get('archives', [])
+
+    if isinstance(archives, basestring):
+        archives = [str(archives)]
+    elif not isinstance(archives, list):
         raise CommandExecutionError(
-            'Invalid products list or string expected and got: {0}'.format(
-                products
+            'Invalid archives: list or string expected, got {0}'.format(
+                archives
             )
         )
 
     res = {}
-
-    for prod in products:
-        if os.path.isdir(prod):
+    for archive in archives:
+        if os.path.isdir(archive):
             iso = None
-            info = product_info_from_tree(prod)
-            path = prod
+            info = archive_info_from_tree(archive)
+            path = archive
             version = info['version']
-        elif os.path.isfile(prod):
-            iso = prod
-            info = product_info_from_iso(prod)
+        elif os.path.isfile(archive):
+            iso = archive
+            info = archive_info_from_iso(archive)
             version = info['version']
             path = '/srv/scality/metalk8s-{0}'.format(version)
```

Comment:
I find the `path` variable name here confusing because from what I understand by reading the code a few times, it's a dir path, but by looking at this 217th line quickly, it could look like a file path. Could we have it renamed `dirpath` for example?